src/main.js

```diff
@@ -22,45 +22,55 @@ const ipc = electron.ipcMain;
  * */
 let mainWindow;
 
+/** NT IDS of strings  */
+let stringIDs = [];
```

Comment:
Why do we need this?

Reply:
That is to keep track of which strings were converted to boolean since they need to be changed back before update can take place due to types being permanent on the NT server.

Reply:
Hmm, okay. I would argue that we should handle that in the NT client (`pynetworktables2js` used to do this, as I recall).